plugins/ShareThis/class.sharethis.plugin.php

```diff
@@ -14,8 +14,8 @@ class ShareThisPlugin extends Gdn_Plugin {
     */
 	public function discussionController_afterDiscussionBody_handler($sender) {
       $publisherNumber = c('Plugin.ShareThis.PublisherNumber', 'Publisher Number');
-      $viaHandle = c('Plugin.ShareThis.ViaHandle', '');
-      $copyNShare = c('Plugin.ShareThis.CopyNShare', false);
+      $viaHandle = htmlspecialchars(c('Plugin.ShareThis.ViaHandle', ''));
+      $copyNShare = htmlspecialchars(c('Plugin.ShareThis.CopyNShare', false));
```

Comment:
```suggestion
      $copyNShare =c('Plugin.ShareThis.CopyNShare', false);
```

This is a boolean and isn't being outputted. My feeling is it will actually break the conditional check as well because it gets converted to a string through `htmlspecialchars`.

Reply:
@charrondev Thanks for catching this.